src/testing.rst

```diff
@@ -29,8 +34,125 @@ On Windows, use the ``exist`` command. See below for an example.
         - if not exist %PREFIX%\\Library\\lib\\boost_log-vc140-mt.lib exit 1  # [win and py35]
 
 
+Testing Python Packages
+-----------------------
+
```

Comment:
The first thing we should do is to reference the conda-build docs on this https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-section.

I'm keen to document what we need to for conda-forge, but for everything else try to push upstream to conda-build.

Heading that I think we should have:

 * Testing imports
 * Running unit tests (pytest)
 * Testing CLIs

Reply:
added link to conda docs.